### DISCUSSION TITLE
When is it necessary to replace and modify the signal of the octal bus transceiver?

### DISCUSSION THREAD

**Discussion by ThomasKaemmerer** [Peter-van-Tol/LiteX-CNC#99] [Q&A] [answered]
Am I right in assuming that the HW modification with the replacement of the existing 74HC245T octal bus transceivers with 74LVC245 is only necessary if the corresponding HUB connectors are to be used as input?

**Comment by Peter-van-Tol** [comment, marked as answer]
Yes this is correct. When the pins are used for output, you can use the original buffer ICs.

If you want to use SPI for connecting the card, you have to replace at least 1 buffer with bridges (no buffer at all),
